File: OrientacionAObjeto/Donativos/Objeto_python/Donante.py
class Donante:

    def __init__(self, nombre, correo, cantidad, recurrencia):
        self._nombre = nombre
        self._correo = correo
        self._cantidad = cantidad
        self._recurrencia = recurrencia

    def __str__(self) -> str:
        return f"Me llamo {self._nombre} y mi correo es {self._correo}"
    
    def getDonativo(self):
        return{'cantidad': self._cantidad, 'recurrencia': self._recurrencia}

File: OrientacionAObjeto/Donativos/Objeto_java/Donante.java
package Objeto_java;

class Donante {
    private String nombre;
    private String correo;
    private Donativo donativo;

    public Donante(String nombre, String correo, Donativo donativo){
        this.nombre = nombre;
        this.correo = correo;
        this.donativo = donativo;
    }

    public String toString(){
        return "Me llamo " + nombre + " y mi correo es " + correo;
    }

    public Donativo getDonativo() {
        return donativo;
    }

    public String getNombre() {
        return nombre;
    }

    public String getCorreo() {
        return correo;
    }

}